backend/RestAPI/Controllers/VideosController.cs

```diff
@@ -33,10 +32,88 @@ public class VideosController : ControllerBase
             _videoService = videoService;
         }
 
-        [HttpGet,Route("all")]
-        public async Task<ActionResult<IEnumerable<Video>>> GetAllVideos()
```

Comment:
This is not neccessary but if you specify the return type with ActionResult, you can see the output example in swagger 
![image](https://user-images.githubusercontent.com/55400760/118478176-3e76c000-b718-11eb-81b9-629966e310b4.png)

Reply:
Will try to do that 👍